generate.go

```diff
@@ -2,6 +2,9 @@ package veneur
 
 //go:generate protoc --gofast_out=Mssf/sample.proto=github.com/stripe/veneur/ssf,plugins=grpc:. sinks/grpsink/grpc_sink.proto
 //go:generate protoc --gofast_out=. ssf/sample.proto
+//go:generate protoc -I=. -I=$GOPATH/src -I=$GOPATH/src/github.com/gogo/protobuf/protobuf --gogofaster_out=. tdigest/tdigest.proto
```

Comment:
These invocations appear to assume that there are proto dependencies placed on GOPATH. If so, while that might be acceptable for the docker build, we can't have it for dev. Gotta encapsulate these into vendor/. `required` entries in `Gopkg.toml` should achieve the desired effect.